bin/start

```diff
@@ -67,5 +67,6 @@ elif [[ $RUN_IN_DOCKER = true ]]; then
   # this option cannot run locally due to lack of docker credentials. It works only in the Jenkins pipeline
   runScriptWithSummon ./test_in_docker.sh
 else
-  runScriptWithSummon ./test_local.sh
+  cd common
+  runScriptWithSummon ./summon/secrets.yml ./run.sh
```

Comment:
do we want to use `pushd` instead? also - why do we have a `common` directory? wouldn't the common files just be in the parent directory of `test` and `dev`?

Reply:
I agree in addition to that you have summon/secrets.yml and common/summon/secrets.yml not clear the different here

Reply:
I will 
- place both flows (dev and test) in common directory
- rename common -> deploy for better naming
- I delete the second summon that was in dev since it is not needed

@eladkug what did you mean here
> Do you have to enter the directories or you can run it directly ./dev/reload.sh ?

Reply:
Instead of getting into the directory by cd common. why not run it directly from current location for example:
instead of:
```
cd common
runScriptWithSummon ./summon/secrets.yml ./run.sh
```
Run:
`runScriptWithSummon ./common/summon/secrets.yml ./run.sh`

Reply:
That changes the scope at which the script will run so it will have cascading effects in the calling scripts. It is something I thought about and attempted but it is a bigger change than I would like to conquer in this PR